### PR TITLE
Add listing screenshots for f-droid

### PR DESCRIPTION
**Pull request by kewisch**
This PR adds various f-droid listing metadata for the upcoming beta and release listing.

### CODE REVIEW

File: app-metadata/net.thunderbird.android.beta/en-US/full_description.txt [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Find our bug tracker, source code, and wiki at <a href="https://github.com/thunderbird/thunderbird-android">https://github.com/thunderbird/thunderbird-android</a>.
+
+We’re always happy to welcome new developers, designers, documenters, translators, bug triagers and friends. Visit us at <a href="https://thunderbird.net/participate">https://thunderbird.net/participate</a> to get started.
+
+What you can do
+Thunderbird is a powerful, privacy-focused email app. Effortlessly manage multiple email accounts from one app, with a Unified Inbox option for maximum productivity. Built on open-source technology and supported by a dedicated team of developers alongside a global community of volunteers, Thunderbird never treats your private data as a product. Supported solely by financial contributions from our users, so you never have to see ads mixed in with your emails again.
+
+<h2>What you can do</h2>
```
Comment (cketti) [inline app-metadata/net.thunderbird.android.beta/en-US/full_description.txt:10]: "What you can do" is present twice. Should this be one heading with a paragraph of text, followed by a list?

File: app-metadata/net.thunderbird.android.beta/en-US/full_description.txt [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  <ul>
+  <li>Ditch multiple apps and webmail. Use one app, with an optional Unified Inbox, to power through your day.</li>
+  <li>Enjoy a privacy-friendly email client that never collects or sells your personal data. We directly connect you to your email provider. That’s it!</li>
+  <li>Take your privacy to the next level by using OpenPGP email encryption (PGP/MIME) with the “OpenKeychain” app, to encrypt and decrypt your messages.</li>
+  <li>Choose to sync your email instantly, at set intervals, or on-demand. However you want to check your email, it’s up to you!</li>
+  <li>Find your important messages using both local and server-side search.</li>
+</ul>
+
+<h2>Compatibility</h2>
```
Comment (cketti) [inline app-metadata/net.thunderbird.android.beta/en-US/full_description.txt:19]: F-Droid doesn't list `<h2>` as being supported. See https://f-droid.org/docs/All_About_Descriptions_Graphics_and_Screenshots/#html-descriptions

File: app-metadata/net.thunderbird.android.beta/en-US/full_description.txt (hunk 1 above)
Comment (kewisch) [reply app-metadata/net.thunderbird.android.beta/en-US/full_description.txt:10]: Yes, one initial set of paragraphs, then multiple lists with a heading. Thank you for catching!

File: app-metadata/net.thunderbird.android.beta/en-US/full_description.txt (hunk 2 above)
Comment (kewisch) [reply app-metadata/net.thunderbird.android.beta/en-US/full_description.txt:19]: Alright, I'll use <b> then. Thanks!

**Review by cketti — APPROVED** [review]

**Pull request merged**